app/pastes/delete-paste/deletePasteHttpRequest.js

```diff
@@ -1,6 +1,7 @@
 class DeletePasteHttpRequest {
     constructor(request) {
         this.id = request.params.pasteId;
+        this.user = request.user;
```

Comment:
If usecase requires only userId, than provide it with id. User object may content much-much more useless data (for particular usecase)